test/common/testinfra.py

```diff
@@ -581,6 +581,10 @@ def update_status(revision, context, last, changes):
                 status = statuses.get(context, None)
                 baseline = BASELINE_PRIORITY
 
+                # modify the baseline slightly to favor older pull requests, so that we don't
```

Comment:
Why not just reverse the array being enumerated by this line:

```
for pull in self.pulls()
```

Wouldn't that have the same effect?